src/app/dashboard/page.tsx

```diff
@@ -1,14 +1,33 @@
-import { isAuthenticated } from '../../utils/auth';
+import Card from "@/components/Card/Card";
+import { getTasks } from "@/utils/tasks";
+
+export default async function Dashbaord() {
+
+  const tasks = await getTasks();
+  console.log(tasks);
 
-export default function Dashbaord() {
   return (
     <section>
       {/* Replace with your content */}
       <div className="bg-white py-6">
         <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
           {/* Replace with your content */}
           <div className="py-4">
-            <div className="border-4 border-dashed border-gray-200 rounded-lg h-96"></div>
+            <div className="border-4 border-dashed border-gray-200 rounded-lg h-96">
+              <div className="grid grid-cols-3 gap-4">
+                {tasks.map((task) => (
+                  <Card
```

Comment:
Please break component to dynamically pass header and content and make specific TaskHeader and TaskContent components

Reply:
I did something i hope its good enough

Reply:
Card should only have 3 props: header, content, footer.